virtual/environment_test.go

```diff
@@ -42,20 +43,24 @@ var (
 		Discovery:                              testDiscovery,
 		CustomHostFns:                          customHostFns,
 		GCActorsAfterDurationWithNoInvocations: testGCActorsAfterDurationWithNoInvocations,
+		Log:                                    slog.New(slog.NewTextHandler(ioutil.Discard)),
```

Comment:
I'd prefer the logs are emitted for tests so when they fail in CI its easy to understand why